lib/plugins/ray_trace.js

```diff
@@ -27,9 +31,16 @@ module.exports = (bot) => {
     while (pos) {
       const block = bot.blockAt(new Vec3(pos.x, pos.y, pos.z))
       // TODO: Check boundingBox of block
-      if (block && block.type !== 0) {
-        block.face = pos.face
-        return block
+      if (liquid) {
+        if (block && block.type !== 0) {
```

Comment:
In what cases do we need to stop in a liquid ?
If instead of `block.type !== 0` we test for `block.boundingBox !== 'empty'`it will include liquids and plants (and also cave air)

Reply:
> In what cases do we need to stop in a liquid ?
> If instead of `block.type !== 0` we test for `block.boundingBox !== 'empty'`it will include liquids and plants (and also cave air)

To collect in a bucket. Is not it?

Reply:
ok. good point